components/admin_console/cluster_settings.jsx

```diff
@@ -83,10 +83,17 @@ export default class ClusterSettings extends AdminSettings {
                     style={style.configLoadedFromCluster}
                     className='alert alert-warning'
                 >
-                    <i
-                        className='fa fa-warning'
-                        title={Utils.localizeMessage('generic_icons.warning', 'Warning Icon')}
-                    />
+                    <FormattedMessage
```

Comment:
Perhaps we could pull code like this out into its own component? We're repeating similar code in a lot of places where we use the same icons, so we could make components like `WarningIcon` and `RefreshIcon` which would allow us to render `<WarningIcon/>` instead of this whole thing everywhere.

Reply:
Sorry, I should've read the previous comments before reviewing this since this is already covered

Reply:
@hmhealey updated.